wandb/sdk/internal/internal_api.py

```diff
@@ -3762,6 +3762,32 @@ def update_artifact_manifest(
             response["updateArtifactManifest"]["artifactManifest"]["file"],
         )
 
+    def update_artifact_metadata(
+        self, artifact_id: str, metadata: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        """Set the metadata of the given artifact version."""
+        mutation = gql(
+            """
+        mutation UpdateArtifact(
+            $artifactID: ID!,
+            $metadata: JSONString,
+        ) {
+            updateArtifact(input: {
+                artifactID: $artifactID,
+                metadata: $metadata,
+            }) {artifact{id}}
```

Comment:
Did black format this this way? I like:
```
updateArtifact(input: {
    artifactID: $artifactID,
    metadata: $metadata,
}) {
    artifact {
        id
    }
}
```
far more

Reply:
I don't black supports any sort of syntax based formatting in literal strings (except maybe doc strings?)

I changed it